test/features/firestore-consent-handlers.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Firestore Consent Handlers
+
+Scenario: Create Consent With Existing Status
+  Given pisp-demo-server
+  When I create a Consent with an existing status
+  Then the server should do nothing
+
+Scenario: Create New Consent
+  Given pisp-demo-server
+  When a new Consent is created
+  Then the server should assign a consentRequestId and a new status in the consent repository
+
+Scenario Outline: Update Consent With <Status> Status
```

Comment:
@lewisdaly @jgeewax @kleyow @sridharvoruganti @chbhernani   @elnyry-sam-k 
`Examples:` are a great improvement in BDD testing :-D